--- gimmie-0.2.1/gimmie_util.py~	2007-01-18 13:10:17.000000000 +0100
+++ gimmie-0.2.1/gimmie_util.py	2007-01-20 16:08:41.000000000 +0100
@@ -625,7 +625,6 @@
     DEFAULT_BOOKMARKS = [
         ("firefox.desktop", "application/x-desktop"),
         ("nautilus.desktop", "application/x-desktop"),
-        (os.path.abspath("README"), "text/plain"),
         ("aim:goim?screenname=\"orphennui\"", "gaim/buddy"),
         ]
     
